server/broadcast.go

```diff
@@ -206,12 +210,19 @@ func selectOrchestrator(n *core.LivepeerNode, params *streamParameters, cpl core
 	var sessions []*BroadcastSession
 
 	for _, tinfo := range tinfos {
-		var sessionID string
-		var balance Balance
-
-		ticketParams := pmTicketParams(tinfo.TicketParams)
+		var (
+			sessionID    string
+			balance      Balance
+			ticketParams *pm.TicketParams
+		)
 
 		if n.Sender != nil {
+			ticketParams = pmTicketParams(tinfo.TicketParams)
+			ratPrice, err := common.RatPriceInfo(tinfo.PriceInfo)
+			if err != nil {
+				return sessions, err
+			}
+			ticketParams.PricePerPixel = ratPrice
```

Comment:
Doesn't seem like we need this - the `PricePerPixel` field isn't actually used by `n.Sender`

Reply:
fixed in 35ac30958477f13a16d30689c8b7e98be39e3c4d